### ISSUE TITLE
With out setting “no_root_squash” option , stack scope volumes fail with “operation not permitted”. 

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#9363]
Rancher server version - 1.6.3

Steps to reproduce the problem:
Deploy rancher-nfs ( version - rancher/storage-nfs:v0.8.3)
Create a service with stack scoped volumes using rancher-nfs volume driiver.

Service instances fail to come up with the following exceptions:
```
Expected state running but got error: Error response from daemon: chown /var/lib/rancher/volumes/rancher-nfs/mystackagain1_foo3_c2c5e: operation not permitted
```
Settings on nfs server:

/etc/exports mount options 
```
/var/nfs    *(rw,sync,no_subtree_check)
```
Permissions on export directory:

```
drwxr-xr-x 13 nobody nogroup 4096 Jul 15 00:07 nfs
```

External volumes works fine with the above settings.
Note - In case of external volume , I have to create a volume from UI  before starting a service with this external volume.

**Comment by deniseschannon** [comment]
Duplicate of https://github.com/rancher/rancher/issues/7334
